Ball of Duty Server/Domain/Maps/Map.cs

```diff
@@ -149,7 +149,8 @@ public Character AddCharacter(string nickname, Specializations specialization)
                     Y = b.Position.Y,
                     Width = b.Width,
                     Height = b.Height,
-                    Id = c.Id
+                    Id = c.Id,
+                    Specialization = (int) specialization
```

Comment:
i would like you get the specialization from the character object in the case that we in future do some stuff with specialization before returning